plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestParquetDecimalScaling.java

```diff
@@ -300,7 +302,11 @@ public void testReadingNonRescalableDecimals(
                 ImmutableList.of(new ParquetDecimalInsert("value", forceFixedLengthArray, precision, scale, values)),
                 writerVersion);
 
-        assertQueryFails(format("SELECT * FROM tpch.%s", tableName), format("Cannot cast DECIMAL\\(%d, %d\\) '.*' to DECIMAL\\(%d, %d\\)", precision, scale, schemaPrecision, schemaScale));
+        @Language("SQL") String query = format("SELECT * FROM tpch.%s", tableName);
+        @Language("RegExp") String expectedMessage = format("Cannot cast DECIMAL\\(%d, %d\\) '.*' to DECIMAL\\(%d, %d\\)", precision, scale, schemaPrecision, schemaScale);
+
+        assertQueryFails(acceleratedParquetReaderEnabled(false), query, expectedMessage);
```

Comment:
Those sessions could just be constants

Reply:
I wanted to apply the session properties on top of the default session from the query runner